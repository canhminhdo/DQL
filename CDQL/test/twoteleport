#!/bin/sh

DIR=test
cd ../ && maude < twoteleport.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/twoteleport.out
diff $DIR/twoteleport.expected $DIR/twoteleport.out
rm $DIR/twoteleport.out