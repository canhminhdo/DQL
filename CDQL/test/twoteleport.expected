==========================================
reduce in TWO-TELEPORT : initQState |= [H(1) ; CX(1,2) ; H(3) ; CX(3,4) ;
    encap(Alice || Bob || Charlie)[c02 <: 0, c02 <: 1, c12 <: 0, c12 <: 1, c43
    <: 0, c43 <: 1, c53 <: 0, c53 <: 1, c02 :> 0, c02 :> 1, c12 :> 0, c12 :> 1,
    c43 :> 0, c43 :> 1, c53 :> 0, c53 :> 1]]P(2,qubitAt(initQState, 0)) /\ P(3,
    qubitAt(initQState, 5)) .
result JudgmentSet: emptyJS
Bye.
